WORKSPACE

```diff
@@ -245,8 +245,10 @@ go_repository(
 
 go_repository(
     name = "com_github_libp2p_go_libp2p",
-    commit = "c1687281a5c19b61ee5e0dc07fad15697c3bde94",  # v0.4.0
+    commit = "5f1136548865820efe1f7ca1a640a56da6482b51",  # v0.4.0
     importpath = "github.com/libp2p/go-libp2p",
+    remote = "https://github.com/nisdas/go-libp2p",
```

Comment:
this branch is 500+ commits behind master, can we submit a PR upstream to enable peer limiting?